components/molecules/PopularCategoryCard.js

```diff
@@ -1,34 +1,45 @@
 import PropTypes from "prop-types";
 import Image from "next/image";
+import { LibraryIcon } from "@heroicons/react/solid";
 
 export function PopularCategoryCard(props) {
   return (
     <div
-      className={`md:shadow-md h-auto min-h-96 w-full rounded-md border-2 pl-3 pr-3 ${
+      className={`flex flex-col items-center md:shadow-md w-40 h-40 rounded-full border-2 hover:shadow-cards bg-white overflow-clip overflow-hidden${
```

Comment:
1. The life bundles have no visual representation when selected.
2. Life Bundles overlap on a specific width. 

![image](https://user-images.githubusercontent.com/77116011/120492369-031af900-c388-11eb-975b-4420328c412c.png)

Reply:
Fixed it so that there is only 1 circle on xxs screens and 2 up to xl then it goes to 4.